rosidl_adapter/rosidl_adapter/parser.py

```diff
@@ -448,36 +448,39 @@ def parse_message_file(pkg_name, interface_filename):
             pkg_name, msg_name, h.read())
 
 
+def extract_file_level_comments(message_string):
+    lines = message_string.splitlines()
+    index = next(
+        (i for i, v in enumerate(lines) if not v.startswith(COMMENT_DELIMITER)), -1)
+    if index != -1:
+        file_level_comments = lines[0:index]
+        other = lines[index:]
+    else:
+        file_level_comments = lines[:]
+        other = []
+    file_level_comments = [line.lstrip(COMMENT_DELIMITER) for line in file_level_comments]
+    return file_level_comments, other
+
+
 def parse_message_string(pkg_name, msg_name, message_string):
-    file_level_ended = False
-    message_comments = []
     fields = []
     constants = []
     last_element = None  # either a field or a constant
+    # replace tabs with spaces
+    message_string.replace('\t', ' ')
```

Comment:
@ivanpauno hmm, shouldn't this be:
```suggestion
    message_string = message_string.replace('\t', ' ')
```
?

Reply:
yes, see https://github.com/ros2/rosidl/pull/632/commits/8cca81404f5ec6767321f5f4940d0fc8311d6167